debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleChangeEventSourceFactory.java

```diff
@@ -60,4 +65,25 @@ public OracleChangeEventSourceFactory(OracleConnectorConfig configuration, Oracl
                 jdbcConfig,
                 streamingMetrics);
     }
+
+    @Override
+    public Optional<IncrementalSnapshotChangeEventSource<? extends DataCollectionId>> getIncrementalSnapshotChangeEventSource(
+            OracleOffsetContext offsetContext,
+            SnapshotProgressListener snapshotProgressListener,
+            DataChangeEventListener dataChangeEventListener) {
+        // Incremental snapshots requires a secondary database connection
+        // This is because Xstream does not allow any work on the connection while the LCR handler may be invoked
+        // and LogMiner streams results from the CDB$ROOT container but we will need to stream changes from the
+        // PDB when reading snapshot records.
+        //
+        // todo: consider adding a hook so that the connection can be lazily opened & closed when we're done with
+        //          performing any and all incremental snapshot operations.
```

Comment:
We can improve upon this over time because I think it would be wise for connectors that don't rely on the same connection for incremental snapshots to open/close the connection on an as-needed basis.  I just wasn't sure if using the newly added `postReadChunk` method was the ideal place for closing the connection.  @jpechane ?